src/main/java/org/jenkinsci/test/acceptance/po/SlavesMixIn.java

```diff
@@ -54,4 +54,12 @@ private String remoteFs(String name) {
 
         return base + name;
     }
+
+    /**
+     * Get the master node of jenkins.
+     * @return The master node.
+     */
+    public DumbSlave getMaster() {
```

Comment:
This is a problem, because than there exist two configure pages for the same `ConfigurablePageObject` (the system configuration and the node configuration). So when I start configuring jenkins I come to the system configuration but I intend to configure the node. I would add a method in `Jenkins` like 
```
public Node getMasterNode() {
    return new Node(this, url("computer/%s/", getName())) {
        @Override
            return Jenkins.this.getName();
        }
    };
}
```
to provide access to the master node. Do you have a better solution?